.prow.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -xeuo pipefail
+
+yum -y install jq
+for jsonfile in *.json; do
+    jq < $jsonfile >/dev/null
```

Comment:
Hi, might be a dumb question, but do you mind elaborating a bit what does this line do?(Does it just check the validity of each json files). I was also confused about the usage of `jq < $file` vs `jq $file`, do you have pointers to look at for usage of `<` in bash? Thanks, =)

Reply:
Exactly! It'll check that the JSON files are valid. Just a simple check that can be used for now until we have more sophisticated PR tests. `<` means feeding the file as the program's standard input: http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO-3.html